pyxdf/pyxdf.py

```diff
@@ -629,6 +629,7 @@ def _jitter_removal(streams, threshold_seconds=1, threshold_samples=500):
     for stream_id, stream in streams.items():
         stream.effective_srate = 0  # will be recalculated if possible
         nsamples = len(stream.time_stamps)
+        stream.segments = []
```

Comment:
👌🏼 catches empty streams and returns [] instead of [0,-1] (when `_jitter_removal` was called, i.e. `dejitter_timestamps=True`